trade-finance-manager-ui/server/controllers/case/cancellation/check-details.controller.post.test.ts

```diff
@@ -87,23 +88,66 @@ describe('postDealCancellationDetails', () => {
       );
     });
 
-    it('adds a successMessage to req.flash', async () => {
-      // Arrange
-      const session = aRequestSession();
+    describe('when effective from date is in the past', () => {
+      it('adds a successMessage to req.flash', async () => {
+        // Arrange
+        const session = aRequestSession();
 
-      const { req, res } = createMocks<PostDealCancellationDetailsRequest>({
-        params: { _id: dealId },
-        session,
-        body: { reason, bankRequestDate, effectiveFrom },
-        flash: flashMock,
+        const { req, res } = createMocks<PostDealCancellationDetailsRequest>({
+          params: { _id: dealId },
+          session,
+          body: { reason, bankRequestDate, effectiveFrom: sub(new Date(), { days: 1 }).valueOf() },
+          flash: flashMock,
+        });
+
+        // Act
+        await postDealCancellationDetails(req, res);
+
+        // Assert
+        expect(flashMock).toHaveBeenCalledTimes(1);
+        expect(flashMock).toHaveBeenCalledWith('successMessage', `Deal ${ukefDealId} cancelled`);
       });
+    });
 
-      // Act
-      await postDealCancellationDetails(req, res);
+    describe('when effective from date is in the present', () => {
```

Comment:
```suggestion
    describe('when effectiveFrom is in the present', () => {
```

With such descriptions anywhere can we use the actual property name please, easier to read and follow. Also great for any future renamings

Reply:
Yes sure, good idea. Have changed and will make a note to do so in future!